doc/lua_api.txt

```diff
@@ -2820,9 +2820,15 @@ and `minetest.auth_reload` call the authentication handler.
 * `get_gen_notify()`: returns a flagstring and a table with the `deco_id`s
 * `minetest.get_mapgen_object(objectname)`
     * Return requested mapgen object if available (see "Mapgen objects")
+* `minetest.get_heat(pos)`
+    * Returns the heat at the position.
+* `minetest.get_humidity(pos)`
+    * Returns the humidity at the position.
+* `minetest.get_biome_id_at_pos(pos)`
+    * Returns the biome id of the biome at that position.
```

Comment:
Or \`nil\` on failure.

For each of these three new functions.

Reply:
Ok.